stackslib/src/net/api/gettenure.rs

```diff
@@ -195,6 +195,10 @@ impl HttpRequest for RPCNakamotoTenureRequestHandler {
 
         Ok(req_contents)
     }
+
+    fn metrics_identifier(&self) -> &str {
+        "/v3/tenures/:block_id"
```

Comment:
Same comment here as the `/v3/blocks` endpoint

Reply:
7c6a3f0e8